src/components/Modal/modal.css

```diff
@@ -0,0 +1,10 @@
+.modal-backdrop {
+    position: fixed;
+    top: 50%;
+    left: 50%;
+    width: 100%;
+    height: 100%;
+    transform: translate(-50%, -50%);
+    background: rgba(0, 0, 0, 0.05);
```

Comment:
Suggestion: UI of modal doesn't look like a modal you could have added a translucent background color to differentiate it from other components

Reply:
sure will add it